src/sage/topology/moment_angle_complex.py

```diff
@@ -0,0 +1,831 @@
+r"""
+Moment-angle complexes
+
+AUTHORS:
+
+- Ognjen Petrov (2023-06-25)
+
+This module implements the basic structure of moment-angle complexes.
```

Comment:
Nearly all of this should be in the `MomentAngleComplex` class-level doc.

Reply:
I was under the impression that this is what the heading of a Sage file should look like, which I found on the developer's guide. I apologize if I misunderstood, should I reorganize this part?